extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/thing/channels.xml

```diff
@@ -246,4 +246,9 @@
 		<description>Play the Line-in of the the Zone Player corresponding to the given UIN</description>
 	</channel-type>
 
+    <channel-type id="sleeptimer" advanced="true">
```

Comment:
Shouldn't you set
* min = 0
* max = 68399s
* step = 1 (don't know if this is the default or the default is undefined and so also smaller steps are allowed)

In the code you state:
```text
sleep time must be >=0 and < 68400s (24h)
```